README.md

```diff
@@ -140,40 +136,32 @@ end
 
 ## "Simple" *is* an Aesthetic
 
-Again, with software, there's no accounting for taste--but it *is* possible to
-dispassionately analyze complexity. Here is Straitjacket's *raison d'etre*.
-It is a belief and an aesthetic, and it's fine if you don't like it:
+Again, with software, it is only possible to dispassionately analyze complexity.
```

Comment:
Again according to the current definition of the complexity in this document, it means that only functionally minimal software can be deemed "not complex"?

Also there appears to be discrepancy in your messaging here since above you say:

> Simple software minimizes its exposure to complexity. This is why the latest iteration of our code is objectively simpler. 

The first sentence in that describes a property of an undefined term (_simple software_) and the second sentence says something is _objectively_ an undefined term. With a mathematicians cap on this cannot be without a definition, but also is contradicted in the following line right here:

> Again, with software, it is only possible to dispassionately analyze complexity.